### DISCUSSION TITLE
Cannot import name 'timer' from 'browser'

### DISCUSSION THREAD

**Discussion by theslothbear** [brython-dev/brython#2427] [Q&A] [answered]
Several modules such as ajax, timer, local and session storage are not available for import from the browser, and the rest work fine.
![image](https://github.com/brython-dev/brython/assets/128232763/52084557-88ac-43e6-a42d-b70308095d19)
How can I solve this?

**Comment by PierreQuentel** [comment, marked as answer]
Sorry to ask, but did you also include **`brython_stdlib.js`** in the HTML page ?
